### PR TITLE
Update runtime, aspnetcore, and extensions versions in a single dependabot PR

### PR DESCRIPTION
**Pull request by eerhardt**
That way we don't get conflicting PRs like

* https://github.com/dotnet/aspire/pull/4897
* https://github.com/dotnet/aspire/pull/4898
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/4911)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -50,6 +43,10 @@ updates:
       OpenTelemetry:
         patterns:
           - "OpenTelemetry.*"
+      NetPlatform:
+        patterns:
+          - "Microsoft.AspNetCore.*"
+          - "Microsoft.Extensions.*"
```
Comment (danmoseley) [inline .github/dependabot.yml:49]: Add "Microsoft.Orleans*" as well?

also feel free to replace/update https://github.com/dotnet/aspire-samples/pull/369

**Review by danmoseley — APPROVED** [review]

**Comment by danmoseley** [comment]
aside, it doesn't matter for what you're doing here, but these are wildcard patterns not regexes. So ".*" matches a literal dot.

File: .github/dependabot.yml (hunk 1 above)
Comment (eerhardt) [reply .github/dependabot.yml:49]: > Add "Microsoft.Orleans*" as well?

Does it ship at the same time on patch Tuesdays? Or do you mean create a new group for Orleans?

**Comment by eerhardt** [comment]
> Thanks for doing this Eric. If preferred, we can change it so that we only subscribe to extensions which is the highest in the dependency tree, and make the rest of the updates use coherence dependencies so they get all bumped and matching in the same PR.

Dependabot supports this? If there is a better way than what I did here, feel free to update it.

**Comment by joperezr** [comment]
No, sorry should've been more clear. We'd be using dependency-flow for these updates (instead of dependabot) and have dependabot ignore these package ids.

**Pull request merged**